src/GuiBase/KeyFrameEditor/KeyFramedValueEditorFrame.h

```diff
@@ -0,0 +1,150 @@
+#ifndef RADUIUMENGINE_QEDITORFRAME_H_
+#define RADUIUMENGINE_QEDITORFRAME_H_
+
+#include <QDoubleSpinBox>
+#include <QFrame>
+#include <QLabel>
+#include <QSpacerItem>
+#include <QToolButton>
+#include <set>
+
+#include <Core/Math/LinearAlgebra.hpp>
+
+namespace Ra::Core::Animation {
+class KeyFramedValueBase;
+}
+
+namespace Ra::GuiBase {
+class QEditorWidgetRuler;
+}
+
+namespace Ui {
+class KeyFramedValueEditor;
+}
+
+namespace Ra::GuiBase {
+class KeyFramedValueEditor;
+
+class QEditorFrame : public QFrame
```

Comment:
Why naming this class as an internal Qt class ? As it is a Radium class, name it according to Radium guidelines and prefer to not introduce confusing names in classes or filename.